### PR TITLE
remove public/private for studies

### PR DESCRIPTION
**Pull request by AbdelHedhili**
Signed-off-by: Abdelsalem <abdelsalem.hedhili@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/StudyController.java [hunk 1]
```diff
@@ -81,17 +81,17 @@ public void initBinder(WebDataBinder webdataBinder) {
...
+    public ResponseEntity<Flux<CreatedStudyBasicInfos>> getStudyList() {
+        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getStudyList());
     }
 
     @GetMapping(value = "/study_creation_requests")
     @Operation(summary = "Get all study creation requests for a user")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The list of study creation requests")})
-    public ResponseEntity<Flux<BasicStudyInfos>> getStudyCreationRequestList(@RequestHeader("userId") String userId) {
-        Flux<BasicStudyInfos> studies = studyService.getStudyCreationRequests(userId);
+    public ResponseEntity<Flux<BasicStudyInfos>> getStudyCreationRequestList() {
```
Comment (JacquesBorsenberger) [inline src/main/java/org/gridsuite/study/server/StudyController.java:93]: do we really want the list of all study creation request ?

Comment (AbdelHedhili) [reply src/main/java/org/gridsuite/study/server/StudyController.java:93]: for maintenance purpose yes

**Review by JacquesBorsenberger — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/StudyService.java [hunk 2]
```diff
@@ -249,12 +246,11 @@ private static CreatedStudyBasicInfos toCreatedStudyBasicInfos(StudyEntity entit
                 .userId(entity.getUserId())
                 .id(entity.getId())
                 .caseFormat(entity.getCaseFormat())
-                .studyPrivate(entity.isPrivate())
                 .build();
     }
 
-    public Flux<CreatedStudyBasicInfos> getStudies(String userId) {
-        return Flux.fromStream(() -> studyRepository.findAllByUserId(userId).stream())
+    public Flux<CreatedStudyBasicInfos> getStudyList() {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/study/server/StudyService.java:252]: Perhaps better getStudies with Flux ?

File: src/main/java/org/gridsuite/study/server/StudyService.java [hunk 3]
```diff
@@ -313,8 +309,9 @@ private static CreatedStudyBasicInfos toCreatedStudyBasicInfos(StudyEntity entit
                 );
     }
 
-    public Mono<StudyInfos> getCurrentUserStudy(UUID studyUuid) {
-        return getStudy(studyUuid).map(StudyService::toStudyInfos);
+    public Mono<StudyInfos> getStudyInfos(UUID studyUuid) {
+        Mono<StudyEntity> studyMono = getStudy(studyUuid);
```
[the remark this answers is not in the document]
Comment (AbdelHedhili) [reply src/main/java/org/gridsuite/study/server/StudyService.java:313]: changed

File: src/main/java/org/gridsuite/study/server/StudyService.java (hunk 2 above)
Comment (AbdelHedhili) [reply src/main/java/org/gridsuite/study/server/StudyService.java:252]: updated

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**